yarn.config.cjs

```diff
@@ -381,12 +382,15 @@ async function workspaceFileExists(workspace, path) {
 }
 
 /**
- * Expect that the workspace has the given field, and that it is a non-null
- * value. If the field is not present, or is null, this will log an error, and
- * cause the constraint to fail.
+ * This function does one of three things depending on the arguments given:
```

Comment:
Not related to this PR, but it had been a while since I'd worked on this file, and this description was confusing.